src/components/habitEditor/habitEditorComponents/HabitEditorStreaks.js

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+const findLongestConseqSubseq = (arr) => {
+	let n = arr.length;
+	let S = new Set();
+	for (let i = 0; i < n; i++) S.add(arr[i]);
```

Comment:
this could simply be let S = [...new Set(arr)] no?

Reply:
it makes an array which does not have functions like .has which are used later on the object